__tests__/components/AppLayout/SidebarItems/SettingsNavigationItem.comprehensive.test.tsx

```
@@ -0,0 +1,229 @@
+const mockUpdateSettingsSuccess = jest.fn().mockResolvedValue({ ok: true });
+
+jest.mock("@/hooks/settings");
+jest.mock("@/lib/services/settings.service");
+
+describe("SettingsNavigationItem - Missing Coverage", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should render error state (lines 70-75)", () => {
+    const { useSettings } = require("@/hooks/settings");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)